pyhomematic/devicetypes/misc.py

```diff
@@ -137,7 +137,7 @@ def ELEMENT(self):
     "HM-PB-4Dis-WM-2": Remote,
     "HMW-IO-4-FM": Remote,
     "HMIP-WRC2": RemoteBatteryIP,
-    "HmIP-WRC2": RemoteBatteryIP,
```

Comment:
Don't remove the support for HmIP-WRC2. 😉

Reply:
Aah...now I see that. I thought it was a double line. Didn't spot the difference between HMIP and HmIP...

You corrected it, I assume or do I need to do anything else?

Reply:
No, you have to add it again on your branch and push the change. 👍

Reply:
Done....